azure-pipelines-PR.yml

```diff
@@ -738,6 +738,32 @@ stages:
             env:
               FSHARP_EXPERIMENTAL_FEATURES: $(_experimental_flag)
             displayName: End to end build tests
+          
```

Comment:
A few limes above, the strategy for `EndToEndBuildTests` is causing issues because there is two of them and this leads to duplicate artifacts.
Since this is not used anyway now, just remove the `strategy` above alltogether, and also remove `FSHARP_EXPERIMENTAL_FEATURES: $(_experimental_flag)` setting.

i.e. remove lines 723-729;736;739